static/src/javascripts-legacy/projects/common/modules/discussion/loader.js

```diff
@@ -249,7 +249,7 @@ Loader.prototype.initToolbar = function() {
 
 Loader.prototype.initRecommend = function() {
     this.on('click', '.js-recommend-comment', function(e) {
-        upvote.handle(e.currentTarget, this.elem, this.user, DiscussionApi, config.switches.discussionAllowAnonymousRecommendsSwitch);
```

Comment:
Passing these last two params is unnecessary as they are singletons and can be directly imported into the `upvote` module